src/Plugin/ParagraphsPastePlugin/Text.php

```diff
@@ -24,4 +25,25 @@ public static function isApplicable($input, array $definition) {
     return !empty(trim($input));
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function formatInput($value, FieldDefinitionInterface $fieldDefinition) {
+
+    if (in_array($fieldDefinition->getType(), [
+      'text',
+      'text_long',
+      'text_with_summary',
+    ])) {
+      return '<p>' . implode('</p><p>', array_filter(explode("\n", $value))) . '</p>';
```

Comment:
I am not happy with this, since it wraps `<p>`-tags regardless. If the input format has ckeditor enabled or uses some kind of html input, that's fine, but what will happen if we would have something like markdown? I think we should give an option to turn that off.